src/ClassApplication/Controllers/ClassController.cs

```diff
@@ -40,11 +41,62 @@ public async Task<ActionResult<string>> CreateClassroom(Classroom classroom)
         /// <param name="id">classroom id</param>
         /// <returns>An array containing a value determined by the parameter</returns>
         [HttpGet("GetClassByID/{id}")]
-        public async Task<ActionResult<string>> GetClassroom(string id)
+
+        public async Task<ActionResult<string>> GetClassroomByID(string id)
         {
-            throw new NotImplementedException();
+
+            FeedIterator<Classroom> iterator = classesContainer.ReadItemAsync<Classroom>(id);

```

Comment:
So here you don't need to make this var a FeedIterator because we should be getting back a Classroom type object.

Reply:
Classroom= classesContainer.ReadItemAsync<Classroom>(id);

will this be a for this line?

Reply:
yes. something like that